src/locales/en.ts

```diff
@@ -552,6 +553,7 @@ export default {
     },
     qrInfo: 'QR INFO',
     error: 'There was an error submitting this transaction. Please try again.',
+    secureModePaymentsDisabled: 'Payments are disabled in Secure Mode',
```

Comment:
Wasn't sure if this key should be defined with snake_case or camelCase (seems like a mix in this file) - happy to change to whatever's best